src/main.js

```diff
@@ -57,7 +57,7 @@ function createWindow() {
     try {
       installExtension(extension);
     } catch (e) {
-      console.log(e);
+      console.error(e);
```

Comment:
Should we add more info here & some color. e.g. console.error(chalk.red(\`[ELECTRON] Extension installation failed: ${e}\`))

Reply:
I like that format, it's consistent with the usage in `scripts/start.js`.

Reply:
:+1: will update